src/test/java/uk/gov/hmcts/reform/civil/service/citizenui/DashboardClaimInfoServiceTest.java

```diff
@@ -205,6 +205,40 @@ void shouldIncludePaymentDateWhenItExists() {
         assertThat(claimsForDefendant.get(2).getPaymentDate()).isEqualTo(DATE_IN_2025.toLocalDate());
     }
 
+    @Test
+    void shouldGetThePartPaymentImmediateValue() {
+        given(caseDetailsConverter.toCaseData(CASE_DETAILS))
+            .willReturn(CaseData.builder()
+                            .applicant1(Party.builder()
+                                            .individualFirstName("Harry")
+                                            .individualLastName("Porter")
+                                            .type(Party.Type.INDIVIDUAL)
+                                            .build())
+                            .respondent1(Party.builder()
+                                             .individualFirstName(
+                                                 "James")
+                                             .individualLastName("Bond")
+                                             .type(Party.Type.INDIVIDUAL)
+                                             .build())
+                            .claimValue(ClaimValue
+                                            .builder()
+                                            .statementOfValueInPennies(
+                                                new BigDecimal("100000"))
+                                            .build())
+                            .respondToAdmittedClaimOwingAmountPounds(new BigDecimal("500"))
```

Comment:
I would put this in a variable so that we compare the expected result if that makes sense

Reply:
yup, moved to  constants